manifests/argocd-applications/argocd-install-application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: argocd-install
+  namespace: argocd
+spec:
+  destination:
+    namespace: argocd
+    name: in-cluster
+  project: default
+  source:
+    path: manifests/argocd-install
+    # repoURL: https://github.com/kuadrant/deployment
```

Comment:
is that once this merges we will update this?

Reply:
Yes, once the main branch has the actual code, we can point all apps to main.